forum/models.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models
+
+class Comment(models.Model):
+    author = models.ForeignKey('student.Student')
+    content = models.TextField()
+    timestamp = models.DateTimeField(auto_now_add=True)
+    transcript_id = models.ForeignKey('forum.Transcript')
```

Comment:
I believe the syntax here is `models.ForeignKey(Transcript)` -- and you have to define the Transcript class before this Comment class since it is being referenced here before Transcript is defined.